test/GithubApi.Users.test.js

```diff
@@ -0,0 +1,63 @@
+const agent = require('superagent');
+const responseTime = require('superagent-response-time');
+const { expect } = require('chai');
+
+const urlApi = 'https://api.github.com';
+const usersTest1 = 10;
+const usersTest2 = 50;
+
+describe('Given a github user', () => {
+  describe('When gets all users', () => {
+    let queryTime;
+
+    before(async () => {
+      await agent
+        .get(`${urlApi}/users`)
+        .auth('token', process.env.ACCESS_TOKEN)
+        .set('User-Agent', 'agent')
+        .use(responseTime((request, time) => {
+          queryTime = time;
+        }));
+    });
+
+    it('Then it should have a response less than 5 seconds', () => {
+      expect(queryTime).to.be.at.below(5000);
+    });
+  });
+
+  describe('When get only 10 users', () => {
+    let users;
+
+    before(async () => {
+      const response = await agent
+        .get(`${urlApi}/users`)
+        .auth('token', process.env.ACCESS_TOKEN)
+        .set('User-Agent', 'agent')
+        .query({ per_page: usersTest1 });
+
+      users = response.body;
+    });
+
+    it('Then the number of filtered users should be equals to 10', () => {
+      expect(users.length).to.equal(10);
```

Comment:
Now that you have a variable, the idea would be to use that variable here instead of the 10